### DISCUSSION TITLE
Command Option

### DISCUSSION THREAD

**Discussion by BryanPattirane** [kangyu-california/PersistentWindows#169] [Q&A]
Hey Kagyu,

Thank you so much for this fix. It's been driving me crazy for a long time. I'm not able to follow one of your instructions to" Specify command option "-delay_start 10" in Edit Action dialog to avoid startup failure after Windows upgrade."

Where is it in the dialog do I do this?

Thank you!

**Comment by kangyu-california** [comment]
This refers to adding a task in Task Scheduler suggested in the main page of PW
here is the dialog to enter command option

![image](https://user-images.githubusercontent.com/59128756/182540474-d2150b74-8a29-4e0a-b41c-49fe748da4f3.png)

**Reply by BryanPattirane** [reply]
And on the dialog box that pops up, where do I enter the "-delay_start 10" ? Is it in the argument box?

![image](https://user-images.githubusercontent.com/8867895/182656723-03ec8a1a-38ba-4b0b-b496-29d339584b4f.png)

**Reply by kangyu-california** [reply]
That's right

**Reply by BryanPattirane** [reply]
Awesome! Thank you so much for creating this solution!

**Comment by kangyu-california** [comment]
"I tried to add a second command, but no matter what I do, only the first command works. How to add multiple commands ?"

Don't create multiple commands, instead you should specify all command options on one command line
-delay_start 10 -silent
